### ISSUE TITLE
Update improvements

### ISSUE THREAD

**Issue by hodasemi** [sharkwouter/minigalaxy#327]
* Adding an option or a button to update everything
* It would be awesome to update all the DLCs when also updating a game. It is super tedious to do it by hand all the time for every single DLC.
  * Maybe this needs to be an option (if not everyone wants this)
  * Another option could be to update all DLCs and also install missing ones

**Comment by sharkwouter** [comment]
Makes sense. Thanks for the report!

**Comment by btegs** [comment]
I'd love to see some sort of Proton integration (if possible) either with our without Steam installed on the system. Also open it up to other versions like Proton-GE.

**Comment by makson96** [comment]
@btegs I think the best place to discuss Proton support is here:
#288

**Comment by sharkwouter** [comment]
@makson96 good point, I forgot about that one. Will close this issue in favor of the older one.

**Comment by sharkwouter** [comment]
This will probably not make it into 1.1.0, so I moved it to 1.2.0.
